From d11acb3c1b29f20972e4d5bb28ea029845cf7cef Mon Sep 17 00:00:00 2001
From: Dinh Nguyen <dinguyen@opensource.altera.com>
Date: Fri, 13 Mar 2015 11:10:26 -0500
Subject: [PATCH 119/172] FogBugz #270904-17: Add 4-bit mode for SD

4-bit SD is now working. The issue was that pinmux config for SD_DAT[4-7]
also need to be configured for SD. The configuration for those pins are
done in u-boot.

Signed-off-by: Dinh Nguyen <dinguyen@opensource.altera.com>
[czou:Original patch taken from
https://github.com/altera-opensource/linux-socfpga.git socfpga-4.1]
Signed-off-by: czou <cao.zou@windriver.com>
---
 arch/arm/boot/dts/socfpga_arria10_socdk_sdmmc.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/socfpga_arria10_socdk_sdmmc.dts b/arch/arm/boot/dts/socfpga_arria10_socdk_sdmmc.dts
index 0d83308..e188cd4 100755
--- a/arch/arm/boot/dts/socfpga_arria10_socdk_sdmmc.dts
+++ b/arch/arm/boot/dts/socfpga_arria10_socdk_sdmmc.dts
@@ -189,7 +189,7 @@
 
 	slot@0 {
 		reg = <0>;
-		bus-width = <1>;
+		bus-width = <4>;
 	};
 };
 
-- 
1.9.1

